graphjet-core/src/main/java/com/twitter/graphjet/algorithms/counting/GeneratorHelper.java

```diff
@@ -53,7 +52,8 @@ private GeneratorHelper() {
         if (socialProof.size() > 1) {
           socialProof.sort();
         }
-        socialProof.trim(maxSocialProofSize);
+
+        socialProof.trim(socialProof.size());
```

Comment:
I want to get rid of this line too. Internally, the capacity of the array grows as a multiple of 2. There are situations like the size of the array is 5, but the capacity is 8. If we do not call trim function here, the following line, new LongArrayList(socialProof.keys()), will generate 3 entries with zero in them. So this line saves clients from trimming zeros on their own.